packages/itwinui-css/src/badge/badge.scss

```diff
@@ -24,7 +24,7 @@ $iui-badge-text-color: rgba($iui-color-foreground-body--light, $iui-opacity-3--l
   color: var(--_iui-badge-text-color);
   text-transform: uppercase;
   user-select: none;
-  max-width: 20ch;
```

Comment:
![image](https://user-images.githubusercontent.com/37936169/172169601-9cb7cd98-8c10-4389-8e0a-bc20735d2ede.png)

Reply:
I am now setting `max-width: calc(20ch + 2px)`.

Also I discovered that non-monospaced fonts will not have exactly 20 characters with `20ch` set. The width allowed for each character is based on the average character width, and if the actual characters are more of the skinnier or more of the wider characters, the number of visible characters may not be what is set.